packages/itwin/breakdown-trees/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/@bentley/build-tools/tsconfig-base.json",
+  "extends":"./node_modules/@itwin/build-tools/tsconfig-base.json",
```

Comment:
any reason for needing to add the types?

Reply:
yes, we reuse some of the configurations from there and removing that will create huge number of build errors.

Reply:
This was here for long time. Just removed it and had no issue.